### DISCUSSION TITLE
Context Community Protocol

### DISCUSSION THREAD

**Discussion by SeanWong24** [ofajs/discussion#24] [Ideas]
Adding support of [Context Community Protocol](https://github.com/webcomponents-cg/community-protocols/blob/main/proposals/context.md) by [Web Components Community Group](https://www.w3.org/community/webcomponents/) may be a good addition to the framework. It defines a standard way to use DOM events to handle contexts data passing between components. 

I noticed that Lit has its [Context](https://lit.dev/docs/data/context/) implementation already.

Maybe related to #9.

**Comment by kirakiray** [comment]
Thank you for your response. Starting from version [4.5.0](https://github.com/kirakiray/ofa.js/releases/tag/4.5.0), ofa.js has already introduced similar functionality. You can use the `o-provider` and `o-context` components to fetch data from the context. For specific usage examples, you can refer to the [test cases directory](https://github.com/kirakiray/ofa.js/tree/main/test/cases/context).

Since the official website is currently undergoing a redesign, the API documentation for this feature has not been updated yet. We will ensure to update it as soon as the website redesign is complete.

**Comment by kirakiray** [comment]
https://ofajs.com/en/documentation/context-state
